server/githubsrm/administrator/models.py

```diff
@@ -58,13 +75,17 @@ def insert_admin(self, doc: Dict[str, str]) -> bool:
             bool
 
         """
-        if self.db.admins.find_one({"email": doc.get("email")}):
-            raise ExistingAdminError("Admin exists")
-        password = doc.pop("password")
-        hash_password = sha256(password.encode()).hexdigest()
+        if self.db.admins.find_one({"email": doc.get('email')}):
```

Comment:
We are using custom errors here. Try to change the function to fit the errors. You should find it in the main branch.